accommodate for more when parsing

kdlslyv opened the pull request:
solves #176, changes are welcome

ndabAP commented:
Awesome work and I love the echo command :)

Just some points:

- It's mandatory to use the conventional commit schema (makes semantic release possible)
- More comments would be nice (e. g.: I'm not sure what "so" stands for)
- Remove the `console.log` please
- Could you run `npm run lint & npm run test` and fix if necessary (I've to fix the CI, it seems)?

kdlslyv commented:
Thanks, I updated the commit

- changed commit message
- had it named `spaceOpts` before, so `so`
- `console.log` removed
- ran the `npm run lint & npm run test`, both fine

ndabAP commented:
Please use this pull request now:

https://github.com/ndabAP/vue-command/pull/197

I wasn't able to directly push to your repository, unfortunately. But your commits are preserver :)
